### PR TITLE
Move Server Azure AD Admin from 2014-04-01 to 2019-06-01-preview

### PR DESCRIPTION
**Pull request by amolagar5**
Move Server Azure AD from 2014-04-01 to 2019-06-01-preview

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/samples/Azure.Messaging.EventHubs.Samples.csproj
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/samples/Azure.Messaging.EventHubs.Samples.csproj]: This file appears to be very out of date.   These changes would cause issues and seem orthogonal to the stated purpose of the PR.   Please revert.

File: sdk/eventhub/Azure.Messaging.EventHubs/tests/Azure.Messaging.EventHubs.Tests.csproj
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/tests/Azure.Messaging.EventHubs.Tests.csproj]: This file appears to be very out of date.   These changes would cause issues and seem orthogonal to the stated purpose of the PR.   Please revert.

**Review by jsquire — DISMISSED** [review]

**Review by jaredmoo — APPROVED** [review]

**Comment by jsquire** [comment]
Thanks!

**Review by isra-fel — COMMENTED** [review]
- Swagger needs to be merged first.
- After it's merged, please regenerate with the spec in the official repo. And make sure the metadata txt is checked in, more [(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)

Thanks

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj
Comment (isra-fel) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj]: The file belongs to AppConfiguration. I don't suppose this change should be checked in.

Comment (isra-fel) [reply sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj]: It's `1.0.0` on master.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @amolagar5 Just one thing: the changes to `sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj` and `tools/PS-VSPrompt.lnk` should not be in this PR. Others look good to me.

Thanks

**Review by isra-fel — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]
@amolagar5  After #10134 was merged master branch had conflict with this PR, mainly in the release notes of csproj file.
I've resolved the conflict, and we just need to wait for the CI to run again.

cc @emgu-ms

**Pull request merged**
